pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -135,6 +143,14 @@ func (vm *windows) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+func (vm *windows) FileExists(path string) (bool, error) {
+	out, err := vm.Run("Test-Path "+path, true)
+	if err != nil {
+		return false, errors.Wrapf(err, "error while checking if the file %s exists on Windows VM %s", path, vm.ID())
+	}
+	return strings.Contains(out, "True"), nil
```

Comment:
Is `True` the only thing returned by the command? We should be as specific as possible when we are parsing. We should not get tripped up if we have a filename that contains `True`.

Reply:
I had preserved the previous logic and just moved it into a function. The command returns "True" or "False" so I agree, `return out == "True"` should suffice.